Makefile

```diff
@@ -10,6 +10,8 @@ SPARK_OPERATOR_DIR := $(ROOT_DIR)/spark-on-k8s-operator
 KONVOY_VERSION ?= v1.1.5
 export KONVOY_VERSION
 
+MKE_CLUSTER_NAME=kubernetes-cluster1
```

Comment:
how about making the name more Spark-specific like `k8s-spark` or something?

Reply:
this name is actually hard-coded in `cluster.sh`, I just introduced separate variable in the Makefile for clarity